test/e2e/e2e_test.go

```diff
@@ -33,6 +35,18 @@ import (
 	"github.com/onsi/gomega"
 )
 
+const (
+	// podStartupTimeout is the time to allow all pods in the cluster to become
+	// running and ready before any e2e tests run. It includes pulling all of
+	// the pods (as of 5/18/15 this is 8 pods).
+	podStartupTimeout = 10 * time.Minute
```

Comment:
Yeah, this check is only run once, before the e2e suite starts; also ditto what Zach said.